### ISSUE TITLE
build failure; fatal error: ffbase/base.h: No such file or directory

### ISSUE THREAD

**Issue by mxmilkiib** [stsaz/ff-3pt#3]
```
...
/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/mac/MAC-ff.cpp: In member function ‘APE::intn APE::CCircleBuffer::RemoveTail(APE::intn)’:
/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/mac/MAC-ff.cpp:129:60: warning: no return statement in function returning non-void [-Wreturn-type]
  129 | APE::intn APE::CCircleBuffer::RemoveTail(APE::intn nBytes){}
      |                                                            ^
g++ -c  -pipe -fvisibility=hidden -O3 -ffunction-sections -fdata-sections -fpic -m64 -march=x86-64 -ISource/Shared -ISource/MACLib -I/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/mac -I/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/../ffos -DPLATFORM_LINUX Source/MACLib/APEDecompress.cpp  -oSource/MACLib/APEDecompress.o
g++ -c  -pipe -fvisibility=hidden -O3 -ffunction-sections -fdata-sections -fpic -m64 -march=x86-64 -ISource/Shared -ISource/MACLib -I/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/mac -I/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/../ffos -DPLATFORM_LINUX Source/MACLib/NewPredictor.cpp  -oSource/MACLib/NewPredictor.o
g++ -c  -pipe -fvisibility=hidden -O3 -ffunction-sections -fdata-sections -fpic -m64 -march=x86-64 -ISource/Shared -ISource/MACLib -I/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/mac -I/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/../ffos -DPLATFORM_LINUX -msse2 Source/MACLib/NNFilter.cpp  -oSource/MACLib/NNFilter.o
g++ -c  -pipe -fvisibility=hidden -O3 -ffunction-sections -fdata-sections -fpic -m64 -march=x86-64 -ISource/Shared -ISource/MACLib -I/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/mac -I/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/../ffos -DPLATFORM_LINUX Source/MACLib/Prepare.cpp  -oSource/MACLib/Prepare.o
g++ -c  -pipe -fvisibility=hidden -O3 -ffunction-sections -fdata-sections -fpic -m64 -march=x86-64 -ISource/Shared -ISource/MACLib -I/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/mac -I/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/../ffos -DPLATFORM_LINUX Source/MACLib/UnBitArrayBase.cpp  -oSource/MACLib/UnBitArrayBase.o
g++ -c  -pipe -fvisibility=hidden -O3 -ffunction-sections -fdata-sections -fpic -m64 -march=x86-64 -ISource/Shared -ISource/MACLib -I/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/mac -I/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/../ffos -DPLATFORM_LINUX Source/MACLib/UnBitArray.cpp  -oSource/MACLib/UnBitArray.o
g++ -c  -pipe -fvisibility=hidden -O3 -ffunction-sections -fdata-sections -fpic -m64 -march=x86-64 -ISource/Shared -ISource/MACLib -I/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/mac -I/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/../ffos -DPLATFORM_LINUX Source/Shared/GlobalFunctions.cpp  -oSource/Shared/GlobalFunctions.o
In file included from /home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/../ffos/FFOS/cpu.h:7,
                 from Source/Shared/GlobalFunctions.cpp:8:
/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/../ffos/FFOS/types.h:34:10: fatal error: ffbase/base.h: No such file or directory
   34 | #include <ffbase/base.h>
      |          ^~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/mac/Makefile:39: Source/Shared/GlobalFunctions.o] Error 1
make[2]: Leaving directory '/tmp/ff3pt-build/mac'
make[1]: *** [/home/milk/state/pkgs/mine/fmedia-git/src/ff-3pt/mac/Makefile:33: all] Error 2
make[1]: Leaving directory '/tmp/ff3pt-build/mac'
make: *** [Makefile:85: mac] Error 2
==> ERROR: A failure occurred in build().
```

**Comment by stsaz** [comment]
Until I fix and test it - consider using `v20.04` tag for `ffos` repo.  The problem is that newer `ffos` requires `ffbase` repo.

**Comment by stsaz** [comment]
Fixed.  Now MAC library doesn't need ffos to build.
